app/assets/stylesheets/application.scss

```diff
@@ -46,6 +46,17 @@ body {
   .container {
     padding-top: 20px;
   }
+  table {
+    width: 350px;
+    text-align: center;
+  }
+  .form {
+    color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.